WIP: Clarify naming for extensions (#8725)
quarkusio/quarkus: docs/src/main/asciidoc/writing-extensions.adoc

TIP: You may want to use the `create-extension` mojo of `io.quarkus:quarkus-maven-plugin` to create these Maven modules - see the next section.

==== Naming

> If your extension are not part of Quarkus it should have a distinct namespace. Do not start your extension group and artifact id with `quarkus`. This is to avoid name clashes between extensions.

Reviewer: ```suggestion
If your extension is not part of Quarkus it should have a distinct namespace. Do not start your extension group and artifact id with `quarkus`. This is to avoid name clashes between extensions.
```

> General rule to follow when doing an external extension is `<your-groupid>.<extension-name>-quarkus`, for example an extension for `ngrok` should be named `io.myext:ngrok-quarkus`.

Reviewer 2: ```suggestion
General rule to follow when doing an external extension is `<your-groupid>:<extension-name>-quarkus`, for example an extension for `ngrok` should be named `io.myext:ngrok-quarkus`.
```

> Where as if `ngrok` would be part of Quarkus it would be named `io.quarkus:quarkus-ngrok`.

Reviewer: ```suggestion
Whereas if `ngrok` would be part of Quarkus it would be named `io.quarkus:quarkus-ngrok`.
```
